cmd/svcat/command/command.go

```diff
@@ -93,3 +109,29 @@ func DetermineNamespace(flags *pflag.FlagSet, currentNamespace string) string {
 
 	return currentNamespace
 }
+
+// AddOutputFlags adds common output flags to a command that can have variable output formats.
+func AddOutputFlags(flags *pflag.FlagSet) {
+	flags.StringP(
+		"output",
+		"o",
+		"",
+		"The output format to use. Valid options are table, json or yaml. If not present, defaults to table",
+	)
+}
+
+func determineOutputFormat(flags *pflag.FlagSet) (string, error) {
+	format, _ := flags.GetString("output")
+	format = strings.ToLower(format)
+
+	switch format {
+	case "", "table":
```

Comment:
Just wanted to agree that handling `""` and `"table"` here instead of setting the default value on the flag is the correct thing to do because in plugin mode, the flag's default value isn't under our control. 👍